docs/administration-guide/databases/mongodb.md

```diff
@@ -1,27 +1,36 @@
 # Working with MongoDB in Metabase
 
+This article covers:
 
-This guide covers:
-
+ - Using a connection string.
  - General connectivity concerns.
  - Connecting to a MongoDB Atlas cluster.
 
+## Using a connection string
+
+If you'd prefer to connect to MongoDB using a [connection string](https://docs.mongodb.com/manual/reference/connection-string/),
+click on **Paste a connection string**. The Metabase UI will update with a field to paste your connection string, as well as present the following options:
```

Comment:
I think we may need to add a comment about the following connection string parameters NOT being supported at this time:

- `tlsCertificateKeyFile`
- `tlsCertificateKeyFilePassword`
- `tlsCAFile`